devicetypes/smartthings/nyce-motion-sensor.src/nyce-motion-sensor.groovy

```diff
@@ -52,7 +52,8 @@ metadata {
 }
 
 def installed() {
-	sendEvent(name: "checkInterval", value: 60 * 12, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+	// device report interval is 0x3600 seconds (230.4 minutes/3.84 hours) so checkinterval is ~that * 2 + 2 minutes
+	sendEvent(name: "checkInterval", value: 2 * 60 * 60 * 4 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
```

Comment:
The +2 minutes doesn't seem like enough to account for a little clock drift. I'd suggest something like 5% or 12 minutes